packages/docs/content/blog/release-9.0.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: Release 9.0
+date: 2024-01-25
+intro: Differentiating between labels, hints, and errors, standardizing dialog close buttons, improving drawers and dialogs, and refining typography design tokens.
+---
+
+import dialogXClose from '../../src/images/dialog-x-close.png';
+
+Along with a host of bugfixes and minor improvements, this major release focused on differentiating between labels, hints, and errors, standardizing dialog close buttons, improving drawers and dialogs, and refining typography design tokens. This release is scheduled for late January, 2024 and is [currently available in beta form](https://github.com/CMSgov/design-system/releases). Read the sections below for more details about this release.
+
+## Whatʼs new in 9.0
+
+### Differentiating between labels, hints, and inline errors
+
+Weʼve restructured form-field labels to improve accessibility, customizability, and clarity of guidance. [Weʼve explained this change in more detail in this migration guide](/migration-guides/v9-label-updates), but the gist is that labels used to contain hints and inline-error messages and now theyʼre separate components. The React components for the various design system form fields handle this change automatically, as they all have labels, hints, and error text built in. Making them separate elements makes them easier to style predictably. Moving the hint and error text out of the HTML `<label>` elements makes them an appropriate place to put interactive content, because assistive technology expects labels to be free of interactive elements. Not having a place to put interactive help resources for a field was a problem, and application teams had no path clear forward. We hope these changes can be leveraged to create even more helpful and accessible forms for delivering important government services.
```

Comment:
Because we have a migration guide we're linking to with more detail, it might be best to simplify this paragraph a little and focus on the impact this change might have on an application. 

> Weʼve restructured form-field labels to improve accessibility, customizability, and clarity of guidance. You can read about this change in more detail in [this migration guide](/migration-guides/v9-label-updates), but the gist is that labels used to contain both hints and inline-error messages and now theyʼre separate components. 
> 
> Making them separate elements makes them easier to style predictably and moving the hint and error text out of the HTML `<label>` elements makes them an appropriate place to put interactive content. 
> 
> The React components for the various design system form fields handle this change automatically, as they all have labels, hints, and error text built in. Unless your application built custom form fields, the impact of this change should be minimal. 
> 
> We hope these changes can be leveraged to create even more helpful and accessible forms for delivering important government services.

Reply:
> Unless your application built custom form fields, the impact of this change should be minimal.

Yeah, that's a better way of saying that